### PR TITLE
build on netbsd

### PR DESCRIPTION
**Pull request by codeck**
tested on raspberry pi

### CODE REVIEW

**Comment by codeck** [comment]
how to?

File: lib/util/easylogging++.h [hunk 1]
```diff
@@ -75,7 +75,7 @@
 #   define _ELPP_OS_FREEBSD 1
 #endif
 // Unix
-#if ((_ELPP_OS_LINUX || _ELPP_OS_MAC || _ELPP_OS_FREEBSD) && (!_ELPP_OS_WINDOWS))
+#if ((_ELPP_OS_LINUX || _ELPP_OS_MAC || _ELPP_OS_FREEBSD || __unix__) && (!_ELPP_OS_WINDOWS))
```
Comment (mvaneijk) [inline lib/util/easylogging++.h]: I don't think you should modify a library file, is this change necessary?

Comment (codeck) [inline lib/util/easylogging++.h]: the better way is make a upstream pull request. but i think it's ok in practical. it's necessary if you want to build on netbsd, which is the only os to support stellar-core on raspberry pi.

**Comment by MonsieurNicolas** [comment]
closing this, I think this was actually fixed when we picked up a new version of easylogging a while back

**Comment by MonsieurNicolas** [comment]
and asio more recently
